docs/proposals/20210526-cluster-class-and-managed-topologies.md

```diff
@@ -138,7 +138,7 @@ command line flags (e.g. via `KubeadmControlPlane`) in the ClusterClass and this
 for all fields of all templates referenced in a ClusterClass.
 
 **Notes**:
-- Only compatible changes (as specified in [ClusterClass compatibility](#clusterclass-compatibility)) should be allowed.
+- Only compatible changes (as specified in [ClusterClass compatibility](#clusterclass)) should be allowed.
```

Comment:
I think this link doesn't make much sense anymore. Same in the other place where we made this change

Reply:
Let's go with: Only compatible changes should be allowed

Reply:
done